README.md

```diff
@@ -39,16 +39,16 @@ cd gem-flow
 
 ### 2. **Enter your creds**
 
-Create a `.env` file in the root directory with your **API keys** and **save** file.
+Create a `.env` file under `<root>/server` directory with your **API keys** and **save** file. A `.env.example` is attached for a full example.
```

Comment:
I had moved the .env file to the root, could move it back, but some files in the serve will need to be updated